README.rst

```diff
@@ -541,6 +554,11 @@ Choose from the list of available fixers:
                         the class name should match
                         the file name. (Risky fixer!)
 
+* **random_api_migration**
```

Comment:
It was late evening in Germany =)

For me it's not problem to support additional replacement options,  e.g. in .php_cs
```php
\Symfony\CS\Fixer\Contrib\RandomApiMigrationFixer::setReplacements(array(
            'rand' => '<whatever>',
            'srand' => '<whatever>',
            'getrandmax' => '<whatever>',
));
```

I can add this If this approach fits fixer's architecture. Any feedback on this?